src/includes/performance-monitoring/configuration/vue.mdx

```diff
@@ -78,10 +89,16 @@ new VueIntegration({
   Vue,
   tracing: true,
   tracingOptions: {
-    trackComponents: ["App", "RwvHeader", "RwvFooter", "RwvArticleList", "Pagination"],
-    hooks: ['mount', 'update', 'destroy']
-  }
-})
+    trackComponents: [
+      "App",
+      "RwvHeader",
+      "RwvFooter",
+      "RwvArticleList",
+      "Pagination",
+    ],
+    hooks: ["mount", "update", "destroy"],
```

Comment:
IDK if a comment in the code here is the right place - perhaps it belongs in the explanatory text above - but it might be nice to mention that specifying the hooks yourself _replaces_ rather than adds to the default list, so `'mount'` and `'update'` need to be in there if they're still going to be tracked.

Reply:
This is a good point, but something we can address in a future PR.